spec/features/report_view_spec.rb

```diff
@@ -41,9 +41,16 @@
 
     let(:workflow_client) { instance_double(Dor::Workflow::Client, workflow_templates: []) }
 
-    it 'returns a page with the expected elements' do
+    it 'returns a page with no get druids from previous search link' do
       visit '/report/bulk'
       expect(page).to have_content('Bulk update operations')
+      expect(page).not_to have_css('.btn-primary', text: 'Get druids from search') # no previous search params, so no get druids button
+      expect(page).to have_css('.btn-primary', text: 'Paste a druid list')
+    end
+
+    it 'returns a page with a get druids from previous search link' do
```

Comment:
This description needs work too

Reply:
Will adjust descriptions to be less awkward